src/logics/compound-v2/configs.ts

```diff
@@ -1,14 +1,365 @@
+      ),
+      cTokens: [
+        new common.Token(
+          common.ChainId.mainnet,
+          '0x041171993284df560249B57358F931D9eB7b925D',
+          8,
+          'cUSDP',
+          'Compound Pax Dollar'
+        ),
+      ],
+    },
+    {
+      underlyingToken: new common.Token(
+        common.ChainId.mainnet,
+        '0xdAC17F958D2ee523a2206206994597C13D831ec7',
+        6,
+        'USDT',
+        'Tether USD'
+      ),
+      cTokens: [
+        new common.Token(
+          common.ChainId.mainnet,
+          '0xf650C3d88D12dB855b8bf7D11Be6C55A4e07dCC9',
+          8,
+          'cUSDT',
+          'Compound USDT'
+        ),
+      ],
+    },
+    {
+      underlyingToken: new common.Token(
+        common.ChainId.mainnet,
+        '0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599',
+        8,
+        'WBTC',
+        'Wrapped BTC'
+      ),
+      cTokens: [
+        new common.Token(
+          common.ChainId.mainnet,
+          '0xccF4429DB6322D5C611ee964527D42E5d685DD6a',
+          8,
+          'cWBTC',
+          'Compound Wrapped BTC'
+        ),
+        new common.Token(
+          common.ChainId.mainnet,
+          '0xC11b1268C1A384e55C48c2391d8d480264A3A7F4',
+          8,
+          'cWBTC',
+          'Compound Wrapped BTC'
+        ),
+      ],
+    },
+    {
+      underlyingToken: new common.Token(
+        common.ChainId.mainnet,
+        '0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e',
+        18,
+        'YFI',
+        'yearn.finance'
+      ),
+      cTokens: [
+        new common.Token(
+          common.ChainId.mainnet,
+          '0x80a2AE356fc9ef4305676f7a3E2Ed04e12C33946',
+          8,
+          'cYFI',
+          'Compound yearn.finance'
+        ),
+      ],
+    },
+    {
+      underlyingToken: new common.Token(
+        common.ChainId.mainnet,
+        '0xE41d2489571d322189246DaFA5ebDe1F4699F498',
+        18,
+        'ZRX',
+        '0x Protocol Token'
+      ),
+      cTokens: [
+        new common.Token(
+          common.ChainId.mainnet,
+          '0xB3319f5D18Bc0D84dD1b4825Dcde5d5f7266d407',
+          8,
+          'cZRX',
+          'Compound 0x'
+        ),
+      ],
+    },
+  ],
 };
 
-type ContractNames = keyof typeof contractAddressMap;
+export const supportedChainIds = [config.chainId];
+
+export const COMP = config.COMP;
+
+export const [underlyingTokens, tokenPairs, underlyingToCTokenMap, cTokenToUnderlyingMap] = config.assets.reduce(
+  (accumulator, { underlyingToken, cTokens }) => {
+    accumulator[0].push(underlyingToken);
+    accumulator[1].push({ underlyingToken, cToken: cTokens[0] });
+    accumulator[2][underlyingToken.address] = cTokens[0];
+    for (const cToken of cTokens) {
+      accumulator[3][cToken.address] = underlyingToken;
+    }
+
+    return accumulator;
+  },
+  [
+    [] as common.Token[],
+    [] as Array<{ underlyingToken: common.Token; cToken: common.Token }>,
+    {} as Record<string, common.Token>,
+    {} as Record<string, common.Token>,
+  ]
+);
```

Comment:
Nit: Is a good idea if we set `mainnetTokens` as `Token[]`, and put both `underlyingToken` and `cToken` to it, and then we can use it in test file like 
```
import { mainnetTokens } from './configs';

input: new common.TokenAmount(mainnetTokens.ETH, '1'),
output: new common.TokenAmount(mainnetTokens.cETH), '0'),
```

Reply:
It seems I misunderstood the usage. My mistake.